aws/vars.tf

```diff
@@ -194,3 +199,15 @@ variable "tiered_storage_enabled" {
   type        = bool
   default     = false
 }
+
+variable "private_key_path" {
+  type        = string
+  description = "The contents of an SSH key to use for the connection. These can be loaded from a file on disk using the file function. This takes preference over password if provided."
+}
```

Comment:
Is this going to force people to set a private_key_path or will the lack of a default here allow this to fall through to the running ssh agent? I guess right now it's implicitly looking for one directly and we're just not setting the variable in a place that people can override easily?

Reply:
Well as far as I can tell we just didn't have a variable for private key. So it threw an error on plan. 

I'm fine making that element optional, but I'm pretty sure it's needed for the ansible connection.

Reply:
We only need to load the public key to make ansible work. I'm only loading a pub key on my builds and relying upon an agent.

Reply:
defaults to null